assets/src/domain/eventEditor/ui/tickets/ticketsList/filterBar/controls/TicketsChainedButton.tsx

```diff
@@ -16,18 +15,18 @@ interface TicketsChainedButtonProps {
  * will appear, otherwise ALL tickets will appear (subject to other filters)
  */
 const TicketsChainedButton: React.FC<TicketsChainedButtonProps> = ({ isChained, toggleIsChained }) => {
+	const className = classNames('ee-filter-bar__btn', {
+		'ee-filter-bar__btn--active': isChained,
+	});
+
+	const icon = isChained ? Icon.LINK : Icon.UNLINK;
 	const tooltip = isChained
 		? __('tickets list is linked to dates list and is showing tickets for above dates only')
 		: __('tickets list is unlinked and is showing tickets for all event dates');
-	const className = isChained
-		? 'ee-filter-bar-filter-btn ee-active-filters ee-ticket-list-is-chained'
-		: 'ee-filter-bar-filter-btn ee-ticket-list-not-chained';
-	const icon = isChained ? Icon.LINK : Icon.UNLINK;
+
 	return (
 		<>
-			<label className='esprs-button-label screen-reader-text' htmlFor={'ee-ticket-list-filter-bar-is-chained'}>
```

Comment:
based on this article https://webaim.org/techniques/forms/controls#button:
`For form buttons (submit and reset input elements and button elements), no additional accessibility information is required.`

Reply:
when a button have text inside that will be used for a11y

Reply:
but when there is no text inside, like in this component it needs to have `aria-label`, this is why I refactored a bit `EspressoButton` to check the `buttonText` and `tooltip`(which is acting like a label)

Reply:
`aria-label` was used based on this recommendation: https://developers.google.com/web/tools/lighthouse/audits/button-name